src/main/java/me/crespel/runtastic/RuntasticExportConverter.java

```diff
@@ -62,19 +75,31 @@ public void run(String[] args) throws Exception {
 	protected void printUsage() {
 		System.out.println("Expected arguments:");
 		System.out.println("  list <export path>");
+		System.out.println("  user <export path>");
 		System.out.println("  info <export path> <activity id>");
+		System.out.println("  photo <exort path> <photo id>");
 		System.out.println("  convert <export path> <activity id | 'all'> <destination> ['gpx' | 'tcx']");
 		System.out.println("  help");
 	}
 
 	protected void doList(File path) throws FileNotFoundException, IOException {
 		SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
-		List<SportSession> sessions = converter.listSportSessions(path);
+		List<SportSession> sessions = converter.listSportSessions(path,false);
 		for (SportSession session : sessions) {
-			System.out.println(sdf.format(session.getStartTime()) + " - ID: " + session.getId() + ", Sport Type: " + session.getSportTypeId() + ", duration: " + Duration.ofMillis(session.getDuration()).toString());
+			System.out.println(sdf.format(session.getStartTime()) + " - ID: " + session.getId() + ", Sport Type: " + session.getSportTypeId() + ", duration: " + Duration.ofMillis(session.getDuration()).toString() + " (" + session.getDuration()/60000 + " min)");
 		}
 	}
 
+	protected void doUser(File path) throws FileNotFoundException, IOException {
+		SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+		User user = converter.getUser(path);
+		System.out.println(sdf.format(user.getCreatedAt()) + " - ID: " + user.getFbUserId() );

```

Comment:
Instead of the Facebook User ID (which may not be filled for everyone), how about showing the Login?
```suggestion
		System.out.println(sdf.format(user.getCreatedAt()) + " - ID: " + user.getLogin() );
```